src/improvements/justfile

```diff
@@ -130,3 +130,17 @@ simulate-all-templates:
         echo "Not all templates files have a corresponding task. Please add a task for a template in ${root_dir}/src/improvements/tasks/example" >&2
         exit 1
     fi
+
+simulate-verify-task TASK="":
```

Comment:
What's the expected usage here? Is this something we want all signers to execute?

Reply:
Yes, that is the expected use case, should I go into modifying `single.just` and `nested.just` along with the documentation?

Or should we merge this as internal use in `src/improvements` and work on the process in a later PR?

Reply:
I'll leave it as is now, and we refactor the signing flow in a separate PR